phylogenetic/rules/export.smk

```diff
@@ -179,3 +185,32 @@ rule final_strain_name:
             --display-strain-name {params.display_strain_field} \
             --output {output.auspice_json}
         """
+
+rule tip_frequencies:
+    """
+    Estimating KDE frequencies for tips
+    """
+    input:
+        tree = "results/{gene}/tree_{serotype}.nwk",
+        metadata = "data/metadata_{serotype}.tsv",
+    output:
+        tip_freq = "auspice/dengue_{serotype}_{gene}_tip-frequencies.json"
+    params:
+        strain_id = config["strain_id_field"],
```

Comment:
> I'm not entirely certain what is causing the empty regions

I was curious what was going on here, so I took a look. There is mismatch in the strain id used in the main JSON and the frequencies JSON so Auspice cannot link the strains to their frequencies. 

Since the Auspice JSON is getting it's strain name set to the `display_strain_field` in `final_strain_name`, the frequencies JSON needs to use the same field here:

```suggestion
        strain_id = config["display_strain_field"],
```

Reply:
Ah, the underlying isse was correct, but the suggestion causes CI to fail. This because the newick tree uses the accession as id so it cannot be matched with the metadata strain. 

Separately discussed with @j23414 to remove `set_final_strain_name.py` as suggested in https://github.com/nextstrain/public/issues/5 so that both use the accession as the strain id.

Reply:
Just logging that @joverlee521  and I chatted that augur frequencies takes `tree.json` file, which is ID'd on `accession` to match against the `metadata.tsv` file. Ergo, it might be time to remove `set_final_strain_name.py` so that both the main JSON and the frequencies JSON use the accession as the strain id.

> This falls inline with [https://github.com/nextstrain/public/issues/5](https://github.com/nextstrain/public/issues/5), see [https://github.com/nextstrain/zika/pull/72](https://github.com/nextstrain/zika/pull/72) as an example

I'll make the changes and re-run tests, thanks!